traffic_ops/traffic_ops_golang/deliveryservice/acme.go

```diff
@@ -219,7 +249,11 @@ func GenerateLetsEncryptCertificates(w http.ResponseWriter, r *http.Request) {
 
 	ctx, _ := context.WithTimeout(r.Context(), AcmeTimeout)
 
-	req := tc.DeliveryServiceLetsEncryptSSLKeysReq{}
+	req := tc.DeliveryServiceAcmeSSLKeysReq{}
+	if req.AuthType == nil {
+		*req.AuthType = tc.LetsEncryptAuthType
```

Comment:
This line will segfault every single time it's run. If `req.AuthType` is `nil`, then dereferencing it like this can only result in a panic. If you want to default to the "Let's Encrypt" auth type, you can do
```go
req.AuthType = new(string)
*req.AuthType = tc.LetsEncryptAuthType
```
(because you can't take the address of a constant, and probably don't want that to be mutable)